### ISSUE TITLE
Line annotation shortcut (ctrl + /) not working since update.

### ISSUE THREAD

**Issue by Omelase** [microsoft/vscode#168384]
Type: <b>Bug</b>

Entering a shortcut removes the original statement and enters '/'.

VS Code version: Code 1.74.0 (5235c6bb189b60b01b1f49062f4ffa42384f8c91, 2022-12-05T16:38:16.075Z)
OS version: Windows_NT x64 10.0.22621
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz (12 x 2592)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|undefined|
|Memory (System)|31.88GB (22.14GB free)|
|Process Argv|--crash-reporter-id 665c2c86-8ece-4100-a423-b1d7c88a1f4f|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (30)</summary>

Extension|Author (truncated)|Version
---|---|---
path-intellisense|chr|2.8.3
vscode-html-css|ecm|1.13.1
EditorConfig|Edi|0.16.4
prettier-vscode|esb|9.10.3
auto-close-tag|for|0.5.14
auto-rename-tag|for|0.1.10
vscode-pull-request-github|Git|0.56.0
live-sass|gle|5.5.1
rest-client|hum|0.25.1
vscode-language-pack-ko|MS-|1.74.12070924
isort|ms-|2022.8.0
python|ms-|2022.20.0
vscode-pylance|ms-|2022.12.20
jupyter|ms-|2022.11.1003412109
jupyter-keymap|ms-|1.0.0
jupyter-renderers|ms-|1.0.12
vscode-jupyter-cell-tags|ms-|0.1.6
vscode-jupyter-slideshow|ms-|0.1.5
cmake-tools|ms-|1.12.27
cpptools|ms-|1.13.6
cpptools-extension-pack|ms-|1.3.0
hexeditor|ms-|1.9.9
color-highlight|nau|2.5.0
indent-rainbow|ode|8.3.1
material-icon-theme|PKi|4.22.0
vscode-css-peek|pra|4.2.0
LiveServer|rit|5.7.9
code-spell-checker|str|2.12.0
cmake|twx|0.0.17
material-theme|zhu|3.15.6

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstes627:30244334
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vsdfh931:30280409
vshan820:30294714
vstes263:30335439
vscoreces:30445986
pythondataviewer:30285071
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593:30376534
pythonvs932:30410667
cppdebug:30492333
vsclangdf:30486550
c4g48928:30535728
dsvsc012:30540252
azure-dev_surveyonecf:30548226
vscccc:30610679
pyindex848:30577860
nodejswelcome1cf:30587006
2e4cg342:30602488
gswce1:30612156
3d0df643:30613357
dbltrim-noruby:30604474
89544117:30613380
fim-prod:30623723
ejf25101:30620729

```

</details>

**Comment by alexdima** [comment]
Let's first try to find out what command executes on your machine, with your settings and your keybindings when pressing `<key>`. Could you please [try the following](https://github.com/microsoft/vscode/wiki/Keybinding-Issues#basic-troubleshooting):

* Run the command `Developer: Toggle Keyboard Shortcuts Troubleshooting`
* This will activate logging of dispatched keyboard shortcuts and will open an output panel with the corresponding log file.
* Press exactly the keybinding you desire.
* check what VS Code detects and what command is invoked.

If you are having trouble understanding the output, please feel free to paste here and I can try to help.

**Comment by tospirits** [comment]
I have a same problem recently.
I thought this problem appeared only when using vscode, but I found a problem that did not input properly in the Windows terminal, so I found a solution.

Did you install Hancom(한글과컴퓨터) software recently? Hancom's input software interfered with the input.

I completely eliminated this problem by deleting the input software from the window settings. I hope it will be solved this way.

Reference : https://gwpaeng.tistory.com/98
